### ISSUE TITLE
test_nblist() is failing in debug mode

### ISSUE THREAD

**Issue by proteneer** [proteneer/timemachine#344]
I've discovered weird error where test_nblist.py passes in ```-O3``` but fails with ```-g -G```. Digging in now....

**Comment by proteneer** [comment]
```
The Volta architecture introduces Independent Thread Scheduling among threads in a warp, enabling intra-warp synchronization patterns previously unavailable and simplifying code changes when porting CPU code. However, this can lead to a rather different set of threads participating in the executed code than intended if the developer made assumptions about warp-synchronicity of previous hardware architectures.
```

**Comment by proteneer** [comment]
Looks like we need to hardcode:

```
-arch=compute_60 -code=sm_70
```

**Comment by proteneer** [comment]
Fixed via #347, which adds the necessary __syncwarps().
